tests/unit/locales-test.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('service:i18n', 'I18nService#locales', {
+  integration: true
+});
+
+const locales = [
+  'en',
+  'en-bw',
+  'en-ps',
+  'en-wz',
+  'es'
+];
+
+test('finds the correct number of locales', function(assert) {
+  const i18n = this.subject();
+
+  assert.equal(i18n.get('locales').length, locales.length);
+});
+
+test('locales have the correct names', function(assert) {
+  const i18n = this.subject();
+
+  assert.expect(locales.length);
+
+  locales.forEach(locale => {
```

Comment:
You could use `assert.deepEqual` here, though you might have to `sort` the results.